### ISSUE TITLE
How to configure timepicker to send moment object in reactive modal while show formatted string on Input filed.

### ISSUE THREAD

**Issue by msudhirsingh** [vlio20/angular-datepicker#506]
Make sure to add **all the information needed to understand the bug** so that someone can help. If the info is missing we'll add the 'Needs more information' label and close the issue until there is enough information.

- [x] Provide a **minimal code snippet**, please make sure it is well formatted.
- [x] Provide a **stackblitz** demo / [stackblitz](https://stackblitz.com/) example that reproduces the bug.
- [ ] Provide **screenshots** where appropriate
- [ ] What's the **version** of Angular you're using?
- [ ] Does this occur on specific browser?

**Comment by vlio20** [comment]
Please provide an example which doesn't work for you. Till then I will close this issue.

**Comment by msudhirsingh** [comment]
Hi @vlio20 , Here is my full explanation about the issue i am facing.
Here is **stackblitz** i created for the issue i am facing.
https://stackblitz.com/edit/angular-tsbjki?file=src%2Fapp%2Fapp.component.html

**[Problem]**
Basically i am using this library in my component as directive with matInput using Reactive froms.
My problem is that after selecting time from picker, my modal is populated with string time value instead of moment object (as in case we use <dp-date-picker> it return moment object in Modal).

So after lots of reading docs and R&D about the issues i am unable to get moment object in my modal after selecting time from picker.
Basically getting a moment object in formcontrol will be me flexibility to format the time in 24 Hours format when user clicks submit buttton to save form, as form data save API requires time in 24 Hours only, but angular side we shows timepicker in 12 or 24 Hours based on preferences.

Please Help and i thank you in advance for your time to fix this issue or possible solution.

**Comment by vlio20** [comment]
hello @msudhirsingh, that makes sense, in directives the value (ngModel) is binded to a string. This the value input. You can get the Moment value via the onSelect output event.

Let me know if it helps
